<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterForm</class>
 <widget class="QWidget" name="RegisterForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>511</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #2e3b4e;
    color: #ffffff;
   
}

QGroupBox {
    background-color: #394a62;
    border-radius: 12px;
    padding: 30px;
    border: none;
}



QLineEdit {
    background-color: #3d4b63;
    color: #ffffff;
    border: 2px solid #5a6b7b;
    border-radius: 8px;
    padding: 10px;
    font-family: 'Segoe UI', sans-serif;
}

QLineEdit:focus {
    border-color: #4CAF50;
    background-color: #2e3b4e;
}

QPushButton {
    background-color: #4CAF50;
    color: white;
    border: 2px solid #4CAF50;
    border-radius: 8px;
    padding: 10px 18px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #45a049;
    border-color: #45a049;
}

QPushButton:pressed {
    background-color: #388e3c;
    border-color: #388e3c;
}

QPushButton#CancelButton {
    background-color: #f44336;
    border-color: #f44336;
}

QPushButton#CancelButton:hover {
    background-color: #e53935;
    border-color: #e53935;
}

QPushButton#CancelButton:pressed {
    background-color: #d32f2f;
    border-color: #d32f2f;
}


QPushButton#RegisterButton {
    background-color: #ff9800;
    border-color: #ff9800;
}

QPushButton#RegisterButton:hover {
    background-color: #fb8c00;
    border-color: #fb8c00;
}

QPushButton#RegisterButton:pressed {
    background-color: #f57c00;
    border-color: #f57c00;
}

QWidget#RegisterForm {
    padding: 30px;
}

QLineEdit {
    background-color: #3d4b63;
    color: #ffffff;
    border: 2px solid #5a6b7b;
    border-radius: 8px;
    padding: 10px;
}

QLineEdit:focus {
    background-color: #2e3b4e;
    border-color: #4CAF50;
}

QPushButton {
    background-color: #4CAF50;
    color: white;
    border: 2px solid #4CAF50;
    border-radius: 8px;
    padding: 10px 18px;
}

QPushButton:hover {
    background-color: #45a049;
    border-color: #45a049;
}

QPushButton:pressed {
    background-color: #388e3c;
    border-color: #388e3c;
}
</string>
  </property>
  <widget class="QGroupBox" name="RegistrationGroupbox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>671</width>
     <height>391</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="title">
    <string>New Member Registration</string>
   </property>
   <widget class="QLabel" name="UsernameLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>VT323</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
font-family:VT323
}</string>
    </property>
    <property name="text">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="UsernameField">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>110</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    background-color: #2e3b4e;
    color: #ffffff;
    font-family: 'Segoe UI', sans-serif;
}

QGroupBox {
    background-color: #394a62;
    border-radius: 12px;
    padding: 30px;
    border: none;
}

QLabel {
    color: #e0e0e0;
    font-weight: bold;
    border: none;
}

QLineEdit {
    background-color: #3d4b63;
    color: #ffffff;
    border: 2px solid #5a6b7b;
    border-radius: 8px;
    padding: 10px;
    font-family: 'Segoe UI', sans-serif;
}

QLineEdit:focus {
    border-color: #4CAF50;
    background-color: #2e3b4e;
}

QPushButton {
    background-color: #4CAF50;
    color: white;
    border: 2px solid #4CAF50;
    border-radius: 8px;
    padding: 10px 20px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #45a049;
    border-color: #45a049;
}

QPushButton:pressed {
    background-color: #388e3c;
    border-color: #388e3c;
}

QPushButton#CancelButton {
    background-color: #f44336;
    border-color: #f44336;
}

QPushButton#CancelButton:hover {
    background-color: #e53935;
    border-color: #e53935;
}

QPushButton#CancelButton:pressed {
    background-color: #d32f2f;
    border-color: #d32f2f;
}

QLabel#UsernameLabel, QLabel#EmailLabel, QLabel#PasswordLabel {
    font-weight: bold;
    border: none;
}

QPushButton#RegisterButton {
    background-color: #ff9800;
    border-color: #ff9800;
}

QPushButton#RegisterButton:hover {
    background-color: #fb8c00;
    border-color: #fb8c00;
}

QPushButton#RegisterButton:pressed {
    background-color: #f57c00;
    border-color: #f57c00;
}

QWidget#RegisterForm {
    padding: 30px;
}

QLineEdit {
    background-color: #3d4b63;
    color: #ffffff;
    border: 2px solid #5a6b7b;
    border-radius: 8px;
    padding: 10px;
}

QLineEdit:focus {
    background-color: #2e3b4e;
    border-color: #4CAF50;
}

QPushButton {
    background-color: #4CAF50;
    color: white;
    border: 2px solid #4CAF50;
    border-radius: 8px;
    padding: 10px 20px;
}

QPushButton:hover {
    background-color: #45a049;
    border-color: #45a049;
}

QPushButton:pressed {
    background-color: #388e3c;
    border-color: #388e3c;
}
</string>
    </property>
   </widget>
   <widget class="QLabel" name="PasswordLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>VT323</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
font-family:VT323
}</string>
    </property>
    <property name="text">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="PasswordField">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::EchoMode::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="EmailLabel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>VT323</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel {
font-family:VT323
}</string>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="EmailTextField">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="RegisterButton">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>320</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Register</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CancelButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>320</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
